localisation/wc_sad.csv

```diff
@@ -156,15 +156,20 @@ write_libram;Write Libram;;Verfasst einen Folianten;;;;;;;;;;;x
+libram_of_retribution_desc;This libram is among the Church of the Holy Light's most ancient relics. He or she who carries it is challenged to become the living embodiment of what this holy time represents.;;;;;;;;;;;;;
 libram_of_holiness;Libram of Holiness;;Foliant der Heiligkeit;;;;;;;;;;;x
-libram_of_holiness_desc;This libram is among the church's most ancient relics. It was given to Saidan Dathrohan by Alonsus Faol, who challenged him to become the living embodiment of what this holy tome represents.;;Dieser Foliant z�hlt zu den �ltesten Relikten der Kirche. Es wurde Saidan Dathrohan von Alonsus Faol gegeben, der ihn dazu aufforderte, die lebendige Verk�rperung dessen zu werden, was dieser heilige Band darstellt.;;;;;;;;;;;x
+libram_of_holiness_key;This libram is among the church's most ancient relics. It was given to [holiness_libram_target.GetBestName] by [libram_giver.GetBestName], who challenged him to become the living embodiment of what this holy tome represents.;;Dieser Foliant z�hlt zu den �ltesten Relikten der Kirche. Es wurde Saidan Dathrohan von Alonsus Faol gegeben, der ihn dazu aufforderte, die lebendige Verk�rperung dessen zu werden, was dieser heilige Band darstellt.;;;;;;;;;;;x
+libram_of_holiness_desc;This libram is among the Church of the Holy Light's most ancient relics. He or she who carries it is challenged to become the living embodiment of what this holy time represents.;;;;;;;;;;;;;
 libram_of_compassion;Libram of Compassion;;Foliant der Barmherzigkeit;;;;;;;;;;;x
-libram_of_compassion_desc;This libram is among the church's most ancient relics. It was given to Gavinrad by Alonsus Faol, who challenged him to become the living embodiment of what this holy tome represents.;;Dieser Foliant z�hlt zu den �ltesten Relikten der Kirche. Es wurde Gavinrad von Alonsus Faol gegeben, der ihn dazu aufforderte, die lebendige Verk�rperung dessen zu werden, was dieser heilige Band darstellt.;;;;;;;;;;;x
+libram_of_compassion_key;This libram is among the church's most ancient relics. It was given to [compassion_libram_target.GetBestName] by [libram_giver.GetBestName], who challenged him to become the living embodiment of what this holy tome represents.;;Dieser Foliant z�hlt zu den �ltesten Relikten der Kirche. Es wurde Gavinrad von Alonsus Faol gegeben, der ihn dazu aufforderte, die lebendige Verk�rperung dessen zu werden, was dieser heilige Band darstellt.;;;;;;;;;;;x
+libram_of_compassion_desc;This libram is among the Church of the Holy Light's most ancient relics. He or she who carries it is challenged to become the living embodiment of what this holy time represents.;;;;;;;;;;;;;
```

Comment:
>He or she

Custom loc can be used here
But I advise to use old descriptions, since they are used in 603 bookmark.

>GetBestName

This also print titles. Imo, GetDynName will be better here.